From 39f89277054a1fde00583299a7fc854f0cba24d1 Mon Sep 17 00:00:00 2001
From: Hongxu Jia <hongxu.jia@windriver.com>
Date: Sat, 29 Jun 2019 16:57:15 +0800
Subject: [PATCH] tweak MAX_TREEINFO_DOWNLOAD_RETRIES

Do not need to retry so many times.

Tweak log debug to log info.

Upstream-Status: Inappropriate [oe specific]

Signed-off-by: Hongxu Jia <hongxu.jia@windriver.com>

Rebase for anaconda 34.

Signed-off-by: Kai Kang <kai.kang@windriver.com>
---
 pyanaconda/payload/install_tree_metadata.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/pyanaconda/payload/install_tree_metadata.py b/pyanaconda/payload/install_tree_metadata.py
index 297ea9ee7..d1fca163e 100644
--- a/pyanaconda/payload/install_tree_metadata.py
+++ b/pyanaconda/payload/install_tree_metadata.py
@@ -28,7 +28,7 @@ from pyanaconda.core.payload import split_protocol
 from pyanaconda.anaconda_loggers import get_packaging_logger
 log = get_packaging_logger()
 
-MAX_TREEINFO_DOWNLOAD_RETRIES = 6
+MAX_TREEINFO_DOWNLOAD_RETRIES = 1
 
 
 class InstallTreeMetadata(object):
-- 
2.7.4

